src/_data/projects.yml

```diff
@@ -33,9 +33,9 @@
    project_description: |
      As part of work on the MEPCOM Integrated Resource System (MIRS), the technical system that underpins the "accessions" process by which recruits are sworn into the military, the Defense Digital Service setup electronic records transfer between the MEPCOM systems and the Army, Navy, and Air Force human resources systems.  This freed recruits from having to carry frequently lost paperwork from the accessions station to boot camp --- as well as keeping staff from having to re-enter the data by hand!  Since launch in late 2017, this system has saved over 1.5 million pages of paper from being carried back and forth.
 
- - title: Army Cyber Pilots
+ - title: Army Cyber Partnership
    img_url: assets/img/layout/army-digital-service.png
    img_description: The Army Digital Service logo; a sphere with a yellow outline, a gold winged image on a black background, ''DEPARTMENT OF DEFENSE'' at top and ''ARMY DIGITAL SERVICE'' below, all white.
    img_credit: USGW
    project_description: |
-     The Defense Digital Service launched a pilot program to help engage and support technical talent within the U.S. Army Cyber Command. Through a series of collaborative pilot projects, highly-trained army officers and soldiers work side-by-side with DDS civilians in unclassified, collaborative, startup-like spaces. Teams tackle real-world problems using technology and tools found in the private sector. Examples of projects include <a href="https://www.wired.com/story/pentagon-dream-team-tech-savvy-soldiers/">developing drone detection technologies</a> for critical missions, hunting adversaries on DoD networks, and <a href="https://www.fedscoop.com/defense-digital-service-redesigns-army-cyber-command-training/">redesigning training for cyber soldiers</a>. 
\ No newline at end of file
+     The Defense Digital Service launched a program to engage and support technical talent within the U.S. Army Cyber Command. Through a series of collaborative projects, highly-trained army officers and soldiers work side-by-side with DDS civilians in unclassified, collaborative, startup-like spaces. Examples of projects include <a href="https://www.wired.com/story/pentagon-dream-team-tech-savvy-soldiers/">developing drone detection technologies</a>, hunting adversaries on DoD networks, and <a href="https://www.fedscoop.com/defense-digital-service-redesigns-army-cyber-command-training/">redesigning training for cyber soldiers</a>.<br><strong>Active Duty Military are encouraged to apply for short-term technical positions under DDS. Military can apply to the 'JYN' program on <a href="https://hire.withgoogle.com/public/jobs/ddsmil/view/P_AAAAAAEAAF3KkxWUnRXoHq">Google Hire</a>.</strong>
```

Comment:
Couple of comments here:

1. 

> highly-trained army officers and soldiers

Shouldn't that be "Army" uppercase?

2. We inconsistently use "DOD" and "DoD" in the copy across the site. Which is most accurate?

3. `'JYN'` should probably be `"JYN"`.

Reply:
These are questions for Mo... I'd ask in Slack (she's not really a GH user)